### DISCUSSION TITLE
Needed to add NSAllowsArbitraryLoads on iOS

### DISCUSSION THREAD

**Discussion by rricamar** [jcesarmobile/ssl-skip#11] [Q&A]
Hello @jcesarmobile, first of all, thanks for the plugin ❤️, it came in handy when working with some dev servers where we cannot change the certificates 😞

I only have a small question: on iOS I had to add the following property on the `info.plist` to make the plugin work (on Android, it works fine with any additional setting):

```xml
<key>NSAppTransportSecurity</key>
<dict>
  <key>NSAllowsArbitraryLoads</key>
  <true/>
</dict>
```

Is this needed or it shouldn't? 🤔 

According to Apple Docs, I thought that this setting only allowed to interact with HTTP requests, so I don't know why the plugin didn't work until I added it...

Thank you in advance 🙏

**Comment by jcesarmobile** [comment]
I've only tested the plugin for live reload servers, in example with angular running `ng serve --ssl` and then setting the Capacitor's `server.url` to `"url": "https://localhost:8100"` (and also with a local IP) and it worked without adding the `NSAppTransportSecurity` entry. I have not tried doing XHR/fetch requests to a `https` url with self signed certificates, is that your use case?

**Reply by rricamar** [reply]
Hello @jcesarmobile,

Yes, that's the case. The server I'm working with is not a local one but a remote one with its own URL.

**Reply by jcesarmobile** [reply]
Do you have the http plugin enabled or disabled?

**Reply by rricamar** [reply]
> Do you have the http plugin enabled or disabled?

Sorry for the late response 😓 

No, I'm not using the Capacitor HTTP plugin or the Cookies plugin 👍
